src/components/RichTextEditor/serializer/nodes/link.ts

```diff
@@ -2,7 +2,7 @@
 
 import { TLinkElement } from '@components/RichTextEditor/Plugins/LinkPlugin/types';
 import { DesignTokens } from '@components/RichTextEditor/types';
-import { TElement } from '@udecode/plate-core';
+import { TElement } from '@udecode/plate';
```

Comment:
Shouldn't we keep `-core` for treeshaking?

Reply:
tree shaking is given for both packages, since both provide ESM

Reply:
it's related to the features used, where many are not part of `@udecode/plate-core` and would bloat the package.json